docs/self-managed/modeler/web-modeler/configuration/configuration.md

```diff
@@ -107,6 +107,27 @@ Refer to the [advanced logging configuration guide](./logging.md#logging-configu
 
 Refer to the [advanced SSL configuration guide](./ssl.md) for additional details on how to set up secure connections (incoming & outgoing) to the Web Modeler components.
 
+### Clusters
```

Comment:
🔧 I think the `Clusters` sub-section can be moved to the top and be the first one under the `Configuration of the restapi component` section as it is an important configuration.

Reply:
It is also worth mentioning that in docker installations, the local zeebe cluster will be pre-configured. Starting with 8.7, we will also provide the pre-configuration in helm installations.

Reply:
👍 Moved this section to the top of the `restapi` section, and added the Docker default. I can add the 8.7 Helm option when that becomes available.